src/main/resources/db/reference/modernised_darts_schema.sql

```diff
@@ -1078,13 +1098,25 @@ IS 'directly sourced from moj_transcription_r';
 COMMENT ON COLUMN transcription_comment.transcription_object_id
 IS 'internal Documentum id from moj_transcription_s acting as foreign key';
 
+CREATE TABLE transcription_document

```

Comment:
Same comment as annotation_document - Do we know what columns should be `NOT NULL` here?